src/style.css

```diff
@@ -1,3 +1,108 @@
+/* Reset de margens padrão do navegador */
+body, h1, h2, h3, h4, h5, h6, p, ul, ol, li, figure, figcaption, blockquote, dl, dd {
+  margin: 0;
+}
+
+/* Importação da fonte Roboto */
 body {
+  background-color: rgb(230, 227, 227);
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  display: flex;
+  justify-content: center; /* Centraliza horizontalmente */
+  align-items: center; /* Centraliza verticalmente */
+}
+
+/* Estilo do header */
+header {
+  width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffffff;
+  padding: 20px;
+  margin-top: 10%;
+}
+
+/* Estilo do logo */
+.logo img {
+  max-height: 40px;
+  order:1;
+  margin-left: 80%;
+}
+
+nav {
+  order: 2; /* Define a ordem da navegação para 2 (o padrão é 0) */
+  margin-right: 5%;
+}
+
+
+/* Estilo do navegador */
+nav ul {
+  list-style-type: none;
   margin: 0;
+  padding: 0;
+}
+
+nav ul li {
+  display: inline;
+  margin-right: 20px;
+  display: inline-block;
+}
+
+nav ul li:last-child {
+  margin-right: 0;
+}
+
+nav ul li a {
+  text-decoration: none;
+  color: #333;
+  text-transform: uppercase;
+  transition: color 0.3s;
+}
+
+nav ul li a:hover {
+  color: #007BFF;
```

Comment:
Use a css variable for the blue color [Reference](https://github.com/mate-academy/layout_moyo-header#:~:text=CSS%20Variable%20is%20used%20for%20a%20blue%20color)